public/scss/_subscriptions.scss

```diff
@@ -0,0 +1,56 @@
+.subscription-list {
+    margin: 0;
+    padding: 0;
+
+    li {
+        list-style: none;
+        margin: 0;
+        padding: 0;
+    }
+}
+
+.subscription {
+    padding-bottom: 2em;
+
+    .product {
+        float: left;
+
+        img {
+            display: block;
+            width: 50px;
+            float: left;
+            margin-right: 10px;
+        }
+
+        .seller {
+            padding: 0;
+            margin: 0 0 0.1em 0;
+            font-weight: normal;
+            float: left;
+            white-space: nowrap;
+            overflow: hidden;
+            text-overflow: ellipsis;
+        }
+
+        .description {
+            text-align: left;
+        }
+    }
+
+    .price {
+        float: right;
+    }
+
+    .subnav {
+        .cancel {
+            float: left;
+        }
+        .change-pay-account {
+            float: right;
+        }
+    }
+
+    .clear {
+        clear: both;
+    }
```

Comment:
I swear I tried `.subnav:after` but it wasn't working. If you can toss in a specific code block I can try it. Knowing css I probably had something wrong but since there are no errors in css (only tears) I'll never know.